app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -471,8 +468,12 @@ class PatientRepository @Inject constructor(
         .toObservable()
   }
 
-  fun addIdentifierToPatient(patientUuid: UUID, identifier: Identifier): Single<BusinessId> {
-    val businessIdStream = createBusinessIdMetaDataForIdentifier(identifier.type)
+  fun addIdentifierToPatient(
+      patientUuid: UUID,
+      identifier: Identifier,
+      loggedInUser: User
```

Comment:
Can we rename `loggedInUser` to be more meaningful in the context of creating business ids?
It could be renamed to `assigningUser`.